cli.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+const spawnSync = require('child_process').spawnSync
+const path = require('path')
+const result = spawnSync(
+  'java',
+  ['-jar', path.join(__dirname, 'compiler.jar')].concat(process.argv.slice(2)),
```

Comment:
Use `COMPILER_PATH` like https://github.com/teppeis/closure-compiler-cli/blob/7607c45ff6860742482fc1afd437f47152ef818d/index.js#L6-L8.

The path may be changed.